dev-python/gst-python/gst-python-1.22.12-r1.ebuild

```diff
@@ -20,6 +20,7 @@ RDEPEND="${PYTHON_DEPS}
 	>=media-libs/gstreamer-${PV}:1.0[introspection]
 	>=media-libs/gst-plugins-base-${PV}:1.0[introspection]
 	>=dev-python/pygobject-3.8:3[${PYTHON_USEDEP}]
+	<dev-python/pygobject-3.52:3
```

Comment:
The move looks weird here 1.24.11 is moved to 1.22.12-r, and 1.22.12 is moved to 1.22.11-r1.